File: msgraph-mail/dotnet/Users/Messages/MultiValueExtendedPropertiesResponse.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
namespace Graphdotnetv4.Users.Messages {
    public class MultiValueExtendedPropertiesResponse {
        public List<MultiValueLegacyExtendedProperty> Value { get; set; }
        public string NextLink { get; set; }
    }
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Users/Item/Messages/Item/MultiValueExtendedProperties/MultiValueExtendedPropertiesResponse.swift
extension Swiftconsoleapp.Users.Item.Messages.Item.MultiValueExtendedProperties {
    public class MultiValueExtendedPropertiesResponse : IAdditionalDataHolder {
        public var additionalData: IDictionary<string, object>
        public var nextLink: String
        public var value: [MultiValueLegacyExtendedProperty]
    }
}
